### DISCUSSION TITLE
12–16 Sept.

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#90] [Progress Report]
🚴🏻 **Progress**

- #85 
- Drafted the demo agenda

😖 **Regressions**

- #88 

💬 **To Discuss**

None

📆 **Next Week**

- Demo
- #88

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**

- Ian out sick. :face_with_thermometer: 

😖 **Regressions**

None

💬 **To Discuss**

- #89 

📆 **Next Week**

- Demo
- #80 
- #67
